cli/pcluster/pcluster.py

```diff
@@ -370,6 +372,31 @@ def stop(args):
         set_asg_limits(asg_name=asg_name, config=config, min=0, max=0, desired=0)
 
 
+def get_version(stack):
+    """
+    Get the version of the stack if tagged.
+
+    :param stack: stack object
+    :return: version or empty string
+    """
+    tags = filter(lambda x: x.get("Key") == "Version", stack.get("Tags"))
+    return tags[0].get("Value") if len(tags) > 0 else ""
+
+
+def colorize(status):
```

Comment:
This is super cool :) I'm just wondering if this specific syntax is incompatible with any shell/os, if so we might want to provide a flag to enable or disable this feature

Reply:
It should work across mac/linux and it looks like windows supports it as of windows 10.

https://en.wikipedia.org/wiki/ANSI_escape_code#Platform_support

Reply:
@enrico-usai Can you test it on windows?

Totally agree on ability to turn it off. Maybe optional cli arg (`--no-color`?)

Reply:
Ok, added `--color` flag